mafContrib/kneMetrics.py

```diff
@@ -155,8 +314,11 @@ def rndm(a, b, g, size=1):
 
     ra, dec = uniformSphere(n_events, seed=seed)
     peak_times = np.random.uniform(low=t_start, high=t_end, size=n_events)
-    file_indx = np.floor(np.random.uniform(low=0, high=n_files, size=n_events)).astype(int)
-    distance = rndm(10, 300, 4, size=n_events)
+    file_indx = np.floor(np.random.uniform(low=0, high=n_files,
```

Comment:
would np.random.randint work for you here instead?

Reply:
Probably yes, but i don't feel 100% confident abut changing this bit (I didn't code it in the first place)